src/errors.rs

```diff
@@ -11,61 +11,80 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use std::error;
-use std::result;
+use grpcio;
+use quick_error::quick_error;
+use std::{error, result};
 
 quick_error!{
```

Comment:
IMO, I want to try Failure instead of quick_error :-)

Reply:
Let me take a look into failure.

Reply:
I also do! :)